### PR TITLE
Jeff's push 

### PR DESCRIPTION
**Pull request by Jeffchiucp**
Hey. I don't like the vagueness of the movie actor list.

### CODE REVIEW

**Comment by Jeffchiucp** [comment]
pull me if you want.

File: ReadMe.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ There is are five different actors, they all interact naturally. That's it. But
 ## Cast
 - Robert Downey Jr as Himself
 - Tony Hawk as Himself
-- Jennifer Lawrence as Herself
-- Ashton Kutcher as Himself
+- Jennifer Lawrence as No thanks man
+- Ashton Kutcher as no Way
```
Comment (tsukudabuddha) [inline ReadMe.md:10]: nah

File: ReadMe.md [hunk 2]
```diff
@@ -6,10 +6,11 @@ There is are five different actors, they all interact naturally. That's it. But
 ## Cast
 - Robert Downey Jr as Himself
 - Tony Hawk as Himself
-- Jennifer Lawrence as Herself
-- Ashton Kutcher as Himself
+- Jennifer Lawrence as No thanks man
```
Comment (tsukudabuddha) [inline ReadMe.md:9]: nah

File: ReadMe.md [hunk 3]
```diff
@@ -6,10 +6,11 @@ There is are five different actors, they all interact naturally. That's it. But
...
+- Ashton Kutcher as no Way
 - Mila Kunis as Herself
 - Bam Margera as Himself
+- I don't like this movie
```
Comment (tsukudabuddha) [inline ReadMe.md]: nah

**Review by tsukudabuddha — CHANGES_REQUESTED** [review]
hmmm.... I think Jennifer lawrence should play herself instead of no thanks man, as that sounds like a very rude character

**Comment by Jeffchiucp** [comment]
looks good to me.

**Pull request closed without merging**